test/e2e/schedulingprofile/intree.go

```diff
@@ -0,0 +1,188 @@
+			gomega.Expect(err).To(gomega.Or(gomega.BeNil(), gomega.Satisfy(apierrors.IsNotFound)))
+		})
+
+		ginkgo.By("Creating propagation policy that references scheduling profile")
+		policy := policies.PropagationPolicyForClustersWithPlacements(f.Name(), clusters[:1])
+		policy.Spec.SchedulingProfile = profile.Name
+		policy, err = f.HostFedClient().
+			CoreV1alpha1().
+			PropagationPolicies(f.TestNamespace().Name).
+			Create(ctx, policy, metav1.CreateOptions{})
+		gomega.Expect(err).ToNot(gomega.HaveOccurred(), framework.MessageUnexpectedError)
+
+		ginkgo.By("Creating configmap that references propagation policy")
+		configMap := resources.GetSimpleConfigMap(f.Name())
+		policies.SetPropagationPolicy(configMap, policy)
+		configMap, err = f.HostKubeClient().CoreV1().ConfigMaps(f.TestNamespace().Name).Create(ctx, configMap, metav1.CreateOptions{})
+		gomega.Expect(err).ToNot(gomega.HaveOccurred(), framework.MessageUnexpectedError)
+
+		ginkgo.By("Verifying scheduling result")
+		if enabled {
+			gomega.Eventually(func(g gomega.Gomega, ctx context.Context) {
+				federatedConfigMap, err := f.HostDynamicClient().
+					Resource(resources.FederatedConfigMapGVR).
+					Namespace(f.TestNamespace().Name).
+					Get(ctx, configMap.Name, metav1.GetOptions{})
+				gomega.Expect(err).To(gomega.Or(gomega.BeNil(), gomega.Satisfy(apierrors.IsNotFound)))
+				g.Expect(err).ToNot(gomega.HaveOccurred())
+
+				placementObj, err := controllerutil.UnmarshalGenericPlacements(federatedConfigMap)
+				gomega.Expect(err).ToNot(gomega.HaveOccurred(), framework.MessageUnexpectedError)
+
+				placement := placementObj.Spec.GetPlacementOrNil(scheduler.PrefixedGlobalSchedulerName)
+				g.Expect(placement).ToNot(gomega.BeNil())
+
+				// only the first cluster should be selected since placement plugin was enabled
+				g.Expect(placement.Clusters).To(gomega.HaveLen(1))
+				g.Expect(placement.Clusters[0].Name).To(gomega.Equal(clusters[0].Name))
+
+				ginkgo.GinkgoLogr.Info("Obtained scheduling result", "result", placement.Clusters)
+			}).WithTimeout(scheduleTimeout).WithContext(ctx).Should(gomega.Succeed(), "Timed out waiting for scheduling")
+		} else {
+			gomega.Eventually(func(g gomega.Gomega, ctx context.Context) {
+				federatedConfigMap, err := f.HostDynamicClient().
+					Resource(resources.FederatedConfigMapGVR).
+					Namespace(f.TestNamespace().Name).
+					Get(ctx, configMap.Name, metav1.GetOptions{})
+				gomega.Expect(err).To(gomega.Or(gomega.BeNil(), gomega.Satisfy(apierrors.IsNotFound)))
+				g.Expect(err).ToNot(gomega.HaveOccurred())
+
+				placementObj, err := controllerutil.UnmarshalGenericPlacements(federatedConfigMap)
+				gomega.Expect(err).ToNot(gomega.HaveOccurred(), framework.MessageUnexpectedError)
+
+				placement := placementObj.Spec.GetPlacementOrNil(scheduler.PrefixedGlobalSchedulerName)
+				g.Expect(placement).ToNot(gomega.BeNil())
+
+				// all clusters should be selected since placement plugin was disabled
```

Comment:
If there's only 1 cluster, will both `enabled` and `!enabled` pass regardless of whether the placement filter is enabled?

Reply:
right, the minimum should be 2. fixed